### ISSUE TITLE
Passing environment variables to ansible

### ISSUE THREAD

**Issue by dlip** [neillturner/kitchen-ansible#189]
I have some playbooks that use lookup to get environment which is on my local pc but not on the instance running test kitchen eg. `{{ lookup('env','SITE_DOMAIN) }}`

I tried using 'env_vars' in the suites section of .kitchen.yml but that passes the variables to serverspec only.

**Comment by neillturner** [comment]
you can use 
  extra_vars to pass a hash of variables. 
or use 
 ansible_playbook_command
to have a custom playbook command where you set the env variables and then run ansible. 

if you to a 
  kitchen converge xxxxxv  -l debug 
you will see the ansible command being used. 

also 
see http://linux.die.net/man/1/ansible-playbook
for the options on an ansible playbook command.

**Comment by dlip** [comment]
If I use extra_vars I need to define a long list of variables when a single environment variable is used in many places. If I use ansible_playbook_command it means I have to manage the ansible command line myself. I still think that adding env_vars is the best approach.

**Comment by neillturner** [comment]
Sorry i don't understand "If I use extra_vars I need to define a long list of variables when a single environment variable is used in many places".  How do environment variables help you over extra_vars?

**Comment by dlip** [comment]
```
extra_vars:
  weba_url: "weba.<%= ENV['SITE_DOMAIN'] %>"
  webb_url: "webb.<%= ENV['SITE_DOMAIN'] %>"
  webc_url: "webc.<%= ENV['SITE_DOMAIN'] %>"
```

vs

```
env_vars:
  SITE_DOMAIN: "<%= ENV['SITE_DOMAIN'] %>"
```

**Comment by neillturner** [comment]
implemented in release 0.45.0
